src/main/java/com/cozyhome/onlineshop/dto/auth/LoginRequest.java

```diff
@@ -17,6 +16,5 @@ public class LoginRequest {
 	@ValidEmail
 	private String username;
```

Comment:
can we rename it to 'email'?  because it's a little bit confused: the different names for validation and field

Reply:
done